pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/AuthenticationTest.java

```diff
@@ -1,44 +1,110 @@
+        patch.setTableAlias("kit");
+
+        Assert.assertTrue(UserUtil.checkGPUserAccessForPatch(realm, gpUserId, null, patch));
+    }
+
+    @Test
+    @Ignore
+    public void GPPatchOtherAccessTest(){
+        String realm = "osteo2";
+
+        String gpUserId = new UserDao().getUserByEmail(cfg.getString(GP_UNIT_TESTER_EMAIL)).orElse(new UserDto()).getId() + "";
+
+        Patch patch = new Patch("0", "participantId", "0", gpUserId , new NameValue("oD.locationPx",  "location"), null, "XSZSRS1MS3D4OAEK2DPM") ;
+        patch.setTableAlias("oD");
+
+        Assert.assertFalse(UserUtil.checkGPUserAccessForPatch(realm, gpUserId, null, patch));
+    }
+
     @Test
-    public void testAuthentication() throws IOException {
-        String auth0Token = "";
-        String userId = "NMRUIZFATUIXIVMHJE9K";
-        String operatorGUID = "C2JYXJUUBC2KP9R10G1U";
-        int cookieAgeInSeconds = 60;
-        //        Map<String, String> claims = new HashMap<>();
-        //        Auth0Util auth0Util = new Auth0Util(cfg.getString(ApplicationConfigConstants.AUTH0_ACCOUNT),
-        //                cfg.getStringList(ApplicationConfigConstants.AUTH0_CONNECTIONS),
-        //                cfg.getBoolean(ApplicationConfigConstants.AUTH0_IS_BASE_64_ENCODED),
-        //                cfg.getString(ApplicationConfigConstants.AUTH0_CLIENT_KEY),
-        //                cfg.getString(ApplicationConfigConstants.AUTH0_SECRET),
-        //                cfg.getString(ApplicationConfigConstants.AUTH0_MGT_KEY),
-        //                cfg.getString(ApplicationConfigConstants.AUTH0_MGT_SECRET),
-        //                cfg.getString(ApplicationConfigConstants.AUTH0_MGT_API_URL),
-        //                false, cfg.getString(ApplicationConfigConstants.AUTH0_AUDIENCE));
-        //         final String clientId = "https://datadonationplatform.org/cid";
-        //         final String userId = "https://datadonationplatform.org/cid";
-        //         final String tenantDomain = "https://datadonationplatform.org/t";
-        //        auth0Util.getClaimValue(auth0Token, tenantDomain).ifPresent( claim -> claims.put(tenantDomain, claim.asString()));
-        //        auth0Util.getClaimValue(auth0Token, clientId).ifPresent( claim -> claims.put(clientId, claim.asString()));
-        //        claims.put("https://datadonationplatform.org/uid", "C2JYXJUUBC2KP9R10G1U");
-        //        String jwtToken = new SecurityHelper().createToken(auth0Token, cookieAgeInSeconds +
-        //        (System.currentTimeMillis() / 1000) + (60 * 5),
-        //                claims);
-
-        Map<String, String> authHeaders = new HashMap<>();
-        authHeaders.put("Authorization", "Bearer " + auth0Token);
-        String reqAddress = "https://pepper-dev.datadonationplatform.org/pepper/v1/user/" + userId + "/studies/cmi-pancan/activities";
-        HttpResponse response = TestUtil.performGet(reqAddress, "", authHeaders).returnResponse();
```

Comment:
This was a test made for FON only, I removed it because FON was cancelled and also in order to clean the code